{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -213,180 +417,107 @@ def bootstrap():
 
 
 @task
-def fetch_db(destination="."):
+@remote
+def dj_migrate_database(c):
     """
-    Dump the database on the remote host and retrieve it locally.
-
-    The destination parameter controls where the dump should be stored locally.
+    Django: Migrate the database
     """
-    require("root")
-
-    dump_path = dump_db("~")
-    get(dump_path, destination)
-    run("rm %s" % dump_path)
-
-    return os.path.basename(dump_path)
+    c.conn.manage_py("migrate")
 
 
 @task
-def import_db(dump_file=None):
+@remote
+def reload_uwsgi(c):
     """
-    Restore the given database dump.
-
-    The dump must be a gzipped SQL dump. If the dump_file parameter is not set,
-    the database will be dumped and retrieved from the remote host.
+    Django: Migrate the database
     """
-    with open("envdir/DATABASE_URL", "r") as db_credentials_file:
-        db_credentials = db_credentials_file.read()
-    db_credentials_dict = dj_database_url.parse(db_credentials)
-
-    if not is_supported_db_engine(db_credentials_dict["ENGINE"]):
-        raise NotImplementedError(
-            "The import_db task doesn't support your database engine"
-        )
-
-    if dump_file is None:
-        dump_file = fetch_db()
-
-    db_info = {
-        "host": db_credentials_dict["HOST"],
-        "user": db_credentials_dict["USER"],
-        "db": db_credentials_dict["NAME"],
-        "db_dump": dump_file,
-    }
-
-    with shell_env(PGPASSWORD=db_credentials_dict["PASSWORD"]):
-        with settings(warn_only=True):
-            local("dropdb -h {host} -U {user} {db}".format(**db_info))
-
-        local("createdb -h {host} -U {user} {db}".format(**db_info))
-        local("gunzip -c {db_dump}|psql -h {host} -U {user} {db}".format(**db_info))
+    c.conn.run_in_project_root(
+        "touch %s"
+        % os.path.join(c.conn.project_root, project_name, "config", "wsgi.py")
+    )
 
 
 @task
-def clean_old_database_backups(nb_backups_to_keep):
-    """
-    Remove old database backups from the system and keep `nb_backups_to_keep`.
-    """
-    backups = ls(get_backups_root())
-    backups = sorted(backups, reverse=True)
-
-    if len(backups) > nb_backups_to_keep:
-        backups_to_delete = backups[nb_backups_to_keep:]
-
-        for backup_to_delete in backups_to_delete:
-            run('rm "%s"' % os.path.join(get_backups_root(), backup_to_delete))
-
-        print("%d backups deleted." % len(backups_to_delete))
-    else:
-        print("No backups to delete.")
-
-
-def is_supported_db_engine(engine):
-    return engine == "django.db.backends.postgresql_psycopg2"
+@remote
+def compile_assets(c):
+    subprocess.run(["npm", "install"])
+    subprocess.run(["npm", "run", "build"])
 
+    subprocess.run(
+        [
+            "rsync",
+            "-r",
+            "-e",
+            "ssh -p {port}".format(port=c.conn.port),
+            "--exclude",
+            "*.map",
+            "--exclude",
+            "*.swp",
+            "static/dist/",
```

Comment:
Remove trailing slash. Otherwise it copies the content of `dist` into `static` (and not the `dist` folder itself)